andela/AndelaEatsUI
# #160283436 Implement add vendor UI

**What does the PR do?**
- Adds `Add Vendor UI`

**Description of Task to be completed?**
- Ensures there is a UI for an Admin to add a vendor.

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git) and
-  cd into AndelaEatsUI and checkout to this branch `ft-create-vendor-modal-UI-160283436`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendors`, and click on `Add Vendor`
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160283436](https://www.pivotaltracker.com/story/show/160283436)

**Screenshots**
`Small screen`
<p align="center">
<img  width="322" alt="screen shot 2018-09-09 at 2 02 15 am" src="https://user-images.githubusercontent.com/26305570/45260636-6e585c00-b3e4-11e8-9a34-db638d6af132.png">
</p>

`Medium Screen`

<p align="center">
<img align="center" width="769" alt="screen shot 2018-09-09 at 2 02 30 am" src="https://user-images.githubusercontent.com/26305570/45260638-74e6d380-b3e4-11e8-98c9-bf15ba96cde9.png">
</p>

`Large Screen`

<img width="1680" alt="screen shot 2018-09-09 at 1 39 07 am" src="https://user-images.githubusercontent.com/26305570/45260639-7d3f0e80-b3e4-11e8-85e9-b9f9c5122610.png">
